<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_login</class>
 <widget class="QMainWindow" name="MainWindow_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园疫情管理系统V0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
	color: rgb(0, 0, 0);
	border: none;
	border-radius:10px
}
QPushButton:hover{
	color:rgb(0, 170, 255)
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
    </property>
    <property name="text">
     <string>没有账号？点我注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;华文行楷&quot;;</string>
    </property>
    <property name="text">
     <string>校园疫情管理系统V0.1</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabWidget::pane{
	border:none;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="student">
     <attribute name="title">
      <string>    学生登陆   </string>
     </attribute>
     <widget class="QLineEdit" name="username_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_stu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
      </property>
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="teacher">
     <attribute name="title">
      <string>    班主任登陆   </string>
     </attribute>
     <widget class="QLineEdit" name="username_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>工号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_tea">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
      </property>
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>核酸检测员登陆</string>
     </attribute>
     <widget class="QLineEdit" name="username_lineEdit_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>工号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(186,186,186);
font: 10pt &quot;微软雅黑&quot;;
border-radius:10px
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_tea_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
      </property>
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>正在连接远程数据库
     请耐心等待......</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
